src/main/kotlin/racingcar/ui/RacingCarController.kt

```diff
@@ -0,0 +1,74 @@
+package racingcar.ui
+
+import racingcar.domain.entity.Car
+import racingcar.domain.usecase.CreateCarsUseCase
+import racingcar.domain.usecase.GetWinnersUseCase
+import racingcar.domain.usecase.PlayRaceUseCase
+
+class RacingCarController(
+    private val racingInputView: RacingInputView,
+    private val racingResultView: RacingResultView,
+    private val createCarsUseCase: CreateCarsUseCase,
+    private val getWinnersUseCase: GetWinnersUseCase,
+    private val playRaceUseCase: PlayRaceUseCase,
+) {
```

Comment:
```suggestion
class RacingCarController(
    private val racingInputView: RacingInputView = RacingInputView(),
    private val racingResultView: RacingResultView = RacingResultView(),
    private val createCarsUseCase: CreateCarsUseCase = CreateCarsUseCase(),
    private val getWinnersUseCase: GetWinnersUseCase = GetWinnersUseCase(),
    private val playRaceUseCase: PlayRaceUseCase = PlayRaceUseCase(),
) {
```
해당 클래스들이 갖는 파라미터가 없을 경우 제안한 것과 같이 컨트롤러 초기값으로 객체를 생성하시면 main() 함수에서 파라미터로 받아오지 않아도 돼서 `RacingCarController().run()`만 이용해 main() 함수에서 컨트롤러를 사용하실 수 있을 것 같아요!

Reply:
main이 앱의 진입점 역할을 해서 객체들을 인스턴스화하는 책임을 가지고 있어야 한다고 생각하는데,
해당 구현은 Controller에서 기초값으로 객체들을 인스턴스화해서 객체들을 인스턴스화 하는 책임이
Controller로 넘어간다고 생각하는데 책임부분을 제외하고 구현하는게 이해하기 편하실까요??

Reply:
말씀대로 main 함수의 객체 생성 책임을 갖는 것이 의존성 주입 원칙에 더 부합하고, 그 역할을 명확히 분리하는 것의 중요성 관점에서는 main에서 주입하는게 더 좋은 방법이네요!

컨트롤러의 초기 설정이 간단하고 코드 간결성을 더 중요시하면 컨트롤러로 책임을 넘겨도 괜찮다고 생각해요.